shaders/linepattern.vertex.glsl

```diff
@@ -8,6 +8,7 @@
 
 attribute vec2 a_pos;
 attribute vec4 a_data;
+attribute vec4 a_color;
```

Comment:
It looks like this (or u_color before it) isn't being used anywhere. Can this be removed?